packages/yoroi-extension/app/styles/context/mode.js

```diff
@@ -29,7 +30,9 @@ function ColorModeProvider({ children, currentTheme }: any): Node {
 
   return (
     <ColorModeContext.Provider value={colorMode}>
-      <ThemeProvider theme={theme}>{children}</ThemeProvider>
+      <CssVarsProvider theme={theme}>
```

Comment:
Do we really need this here? 🤔 We didn't have the package previously and I think for the revamp we can use the theme tokens instead of css vars.